requirements.txt

```diff
@@ -1,4 +1,5 @@
 jinja2
 numpy
+polars
```

Comment:
Would require a documentation update.

Reply:
This is primarily to get the CI working. I'll revert later

Reply:
But it will remain a requirement to execute the tests, right? In the past, we also had tests for the mapping-tester here, which has optional dependencies. Then, however, users install the software, run `ctest` and see that tests are failing.
How does this behave here then?

Reply:
Currently, the `mapping-tester`-test needs `precice-profiling analyze` which requires `polars` anyway. So the dependency is already there.

If we merge #210, then the timings and thus the dependency on polars becomes optional.
In that case, it may be an idea to implement the compare without polars.

On the flip side, this then becomes a bad test, as it tests two different things depending on the environment. Maybe a warning in CMake is enough here.